cmd/rookflex/cmd/mount.go

```diff
@@ -174,3 +181,53 @@ func mount(client *rpc.Client, mounter *k8smount.SafeFormatAndMount, globalVolum
 	}
 	return err
 }
+
+func mountCephFS(client *rpc.Client, opts *flexvolume.AttachOptions) error {
+
+	log(client, fmt.Sprintf("mounting ceph filesystem %s on %s", opts.VolumeName, opts.MountDir), false)
+
+	if opts.ClusterName == "" {
+		return fmt.Errorf("Rook: Attach filesystem %s failed: cluster is not provided", opts.VolumeName)
+	}
+
+	// Get client access info
+	var clientAccessInfo model.ClientAccessInfo
+	err := client.Call("FlexvolumeController.GetClientAccessInfo", opts.ClusterName, &clientAccessInfo)
+	if err != nil {
+		log(client, fmt.Sprintf("Attach filesystem %s on cluster %s failed: %v", opts.VolumeName, opts.ClusterName, err), true)
+		return fmt.Errorf("Rook: Attach filesystem %s on cluster %s failed: %v", opts.VolumeName, opts.ClusterName, err)
+	}
+	devicePath := strings.Join(clientAccessInfo.MonAddresses, ",") + ":/"
```

Comment:
What happens when I want to mount a specific path?
When using the CephFS volume plugin the mount looks like this:
```
1.1.1.1:6790,2.2.2.2:6790,3.3.3.3:6790:/example/application on /data type ceph (rw,seclabel,relatime,name=admin,secret=<hidden>,acl)
```

An example volumes CephFS entry I use right now:
```
      volumes:
      - name: image-store
        cephfs:
          monitors:
          - 1.1.1.1:6790,2.2.2.2:6790,3.3.3.3:6790
          user: admin
          path: "/example/application"
          secretRef:
            name: rook-admin
```

Would it be possible to add `path` as a feature here?

Reply:
ahhh i didnt know about this `path` feature. Ok i will add it. Thanks